test/Table_Tests/src/Database/Postgresql_Spec.enso

```diff
@@ -0,0 +1,82 @@
+from Standard.Base import all
+import Standard.Base.System.Environment
+
+from Standard.Database import all
+from Standard.Database.Connection.Connection import Sql_Error
+import Standard.Test
+import Standard.Table as Materialized_Table
+import project.Database.Common_Spec
+import project.Common_Table_Spec
+import project.Aggregate_Spec
+
+postgres_specific_spec connection pending =
+    Test.group "[PostgreSQL] Info" pending=pending <|
+        connection.execute_update 'CREATE TABLE "Tinfo" ("strs" VARCHAR, "ints" INTEGER, "bools" BOOLEAN, "reals" REAL, "doubles" DOUBLE PRECISION)'
```

Comment:
Can we make a unique table name - this will have issues when we run on multiple threads against a common DB

Reply:
Fair - will improve this as part of next PR.